pkg/cmd/tasks/initcmd/scratch.go

```diff
@@ -59,42 +62,46 @@ func initFromScratch(cfg config) error {
 		return err
 	}
 
+	if err := writeRuntimeFiles(def, scaffolder); err != nil {
+		return err
+	}
+
 	logger.Log(`
 A skeleton Airplane task definition for '%s' has been created in %s! Fill it out with the rest of your task details.
 
 Once you are ready, deploy it to Airplane with:
```

Comment:
Since it's a full skeleton now:

```suggestion
A skeleton Airplane task for '%s' has been created in %s!

Once you are ready, deploy it to Airplane with:
```